backend/fetchArrivals.ts

```
@@ -0,0 +1,107 @@
+import axios from 'axios';
+export type ArrivalInfo = {
+    stationName: string;
+    lineName: string;
+    destinationName: string;
+    timeToStationMinutes: string;
+    towards: string;
+}
+
+type stopPoint = {
+    naptanId: string;
+    distance: number;
+}
+
+export async function getArrivalsGivenPostCode(postCode: string): Promise<ArrivalInfo[][] | string | null> {

```

Quite a meaty return type - you might want to throw errors rather than returning a string